commit/core.pyx

```diff
@@ -879,28 +916,36 @@ cdef class Evaluation :
         # Configuration and results
         print( '\t* Configuration and results:' )
 
-        if save_opt_details:
-            print( '\t\t- results.pickle... ', end='' )
-            sys.stdout.flush()
-            with open( pjoin(RESULTS_path,'results.pickle'), 'wb+' ) as fid :
-                pickle.dump( [self.CONFIG, self.x, x], fid, protocol=2 )
-            print( '[ OK ]' )
+        print( '\t\t- streamline_weights.txt... ', end='' )
+        sys.stdout.flush()
+        xic, _, _ = self.get_coeffs()
+        if stat_coeffs != 'all' and xic.size > 0 :
+            xic = np.reshape( xic, (-1,nF) )
```

Comment:
This is the previous idea, 
the size of `dictionary_TRK_kept` should be used instead of `nF`

Reply:
You're right! Now this should be fixed